### DISCUSSION TITLE
RTL connecting error Dial Proxy Failed,Ttl Expired

### DISCUSSION THREAD

**Discussion by Mahdid42** [Ride-The-Lightning/RTL#1152] [Troubleshooting Questions]
hello, I'm a newbie in this Lightning node and I've been doing a tutorial on how to do a RTL node and other stuff
now I got to the error that I cant solve it based on the answers on internet
I've been trying to connect to a peer but I get this massage with any peer
can you please guide me trough the problem
and Im using raspberry pi 4 and the umbrel os
error text 

Dial Proxy Failed: Socks Connect Tcp 10.21.21.11:9050->i264x6snpgyybqvx7p6kbrsjl5dqypsg23e2gy4o4zbgmrxwqw6qydqd.onion:9735: Unknown Error Ttl Expired

here is the screen shot

![image](https://user-images.githubusercontent.com/45717855/208256383-9b380bf3-e4a3-47b6-b173-abf71c63529d.png)

**Comment by ShahanaFarooqui** [comment]
You are already able to run RTL. This is not an error thrown by RTL. This error is thrown by the peer you are typing to connect with.

**Comment by Mahdid42** [comment]
> You are already able to run RTL. This is not an error thrown by RTL. This error is thrown by the peer you are typing to connect with.

can it be my internet which my node is connected to ?
what causes this error?

**Reply by saubyk** [reply]
Hi @Mahdid42 have you tried connecting with other nodes?
All all the nodes throwing the same error?

Also, please paste the peer uri which you are trying to connect with here

**Reply by Mahdid42** [reply]
> Hi @Mahdid42 have you tried connecting with other nodes? All all the nodes throwing the same error?
> 
> Also, please paste the peer uri which you are trying to connect with here

I did tried other nodes and here is some errors

https://amboss.space/node/03746b93e0b94463624bb28eafa38f3d71b4c80df61a0f10c6340b54f56675487c

03746b93e0b94463624bb28eafa38f3d71b4c80df61a0f10c6340b54f56675487c@refqwpk54vgzukvfdf74ml7txxnu4rrmz46sbuhidwobfbfsltj
zufyd.onion:9735

![image](https://user-images.githubusercontent.com/45717855/208281899-08a6d7b7-b5e3-4dd3-85ef-2fed7e835e63.png)

https://amboss.space/node/026efccf6315aa453f65082c6b8e228d4cce02bf15c6d8495a189019786836c004

026efccf6315aa453f65082c6b8e228d4cce02bf15c6d8495a189019786836c004@i264x6snpgyybqvx7p6kbrsjl5dqypsg23e2gy4o4zbgmrxwqw6qydqd.onion:9735

![image](https://user-images.githubusercontent.com/45717855/208281965-7848abe0-f812-4887-a302-47a2134c9d27.png)

https://amboss.space/node/0250baf7a558091eb9c93f43d595b795db61bd2b55ca016d8682fd310cb1b81e6c

0250baf7a558091eb9c93f43d595b795db61bd2b55ca016d8682fd310cb1b81e6c@46.101.122.181:17759

![image](https://user-images.githubusercontent.com/45717855/208282048-e013efef-09bd-46d5-bbe1-051257c1449a.png)

https://amboss.space/node/037659a0ac8eb3b8d0a720114efc861d3a940382dcfa1403746b4f8f6b2e8810ba

037659a0ac8eb3b8d0a720114efc861d3a940382dcfa1403746b4f8f6b2e8810ba@34.78.139.195:9735
![image](https://user-images.githubusercontent.com/45717855/208282378-081c4dac-038d-43ea-a0a6-978165fa6295.png)

https://amboss.space/node/03b7884a508ecfa7267e365b5caf6f29da16aab3191539282a0a0efe7416fc5f0b

03b7884a508ecfa7267e365b5caf6f29da16aab3191539282a0a0efe7416fc5f0b@ntud2dazne2k6oc7vuevyorjaf63azgnzvclabzbdawt737fzxnzc6ad.onion:9735

![image](https://user-images.githubusercontent.com/45717855/208282467-0665a10a-f6ad-4553-ae67-2395eed3d39d.png)

**Reply by saubyk** [reply]
@Mahdid42 I was able to connect with all of these nodes without any issues on my umbrel
Appears to be an issue with your tor circuit. You can try restarting your device and see if you're able to connect after

**Reply by Mahdid42** [reply]
> @Mahdid42 I was able to connect with all of these nodes without any issues on my umbrel Appears to be an issue with your tor circuit. You can try restarting your device and see if you're able to connect after

okay thanks for the assistance 
I'll try reinstalling the umbrel
let you know if it helped

**Reply by saubyk** [reply]
> okay thanks for the assistance I'll try reinstalling the umbrel let you know if it helped

Try restarting before the extreme step of reinstall
